config/docs/site_processing_config_README.md

```diff
@@ -0,0 +1,167 @@
+e.g.*    
+    - ImageNumber  
+    - ObjectNumber
+
+`control_barcodes:`*The name/s used for non-targeting sgRNA's.    
+e.g.*  
+    - NT
+
+`ignore_files:`*Ignore any files with these names. List the complete file name. Default is .DS_Store because Macs make hidden files with that name.  
+e.g.*  
+    - .DS_Store
+
+## prefilter:
+`perform:` *Do you want to perform 0.prefilter-features step? Set to* true *or* false.
+
+`example_site:` *The name of a single example folder containing data from a single site that is found in project_dir/project/batch.  
+e.g.* CP074A_A1-Site_1  
+*e.g.* 151B2-B1-45
+
+`output_basedir:` *Name the folder where you would like this step to output files. e.g.* data
+
+`flag_cols:` *Searches every feature name and if it finds these it flags them. Flagged files are ignored during 0.prefilter-features.py though they may be used later on.  
+e.g.*  
+    - Barcode  
+    - Location  
+    - Count  
+    - Resize *(Resized images are used for visual output and not measurements.)*
+
+## process-spots:
+`perform:` *Do you want to perform 1.process-spots step? Set to* true *or* false.
+
+`output_basedir:` *Name the folder where you would like this step to output files. e.g.* data
+
+`barcode_cols:` *The column that contains the barcodes (or columns that contain the barcodes) assigned to a given spot.  
+e.g.*  
+    - Barcode_MatchedTo_Barcode
+
+`gene_cols:` *The column that contains the gene (or columns that contain the genes) assigned to a given spot.  
+e.g.*  
+    - Barcode_MatchedTo_GeneCode
+
+`location_cols:` *The columns that define the location of each spot (generally X and Y coordinates).  
+e.g.*  
+    - Location_Center_X  
+    - Location_Center_Y
+
+`spot_score_cols:` *The column that contains the match score for the barcode (or columns that contain the match score for the barcodes) assigned to a spot.  
+e.g.*  
+    - Barcode_MatchedTo_Score
+
+`foci_cols:`*The columns that contain the top match for barcode and corresponding gene.
+e.g.*  
+    - Barcode_BarcodeCalled  
+    - Barcode_MatchedTo_ID
+
+`cell_filter:`*Select the categories of cells that you wish to use for analysis. Categories are defined in cell_quality_utils.py. Make sure that the options you set here are available in the categorize_cells method that you set above in core config.  
+Typical e.g.*  
+    - Perfect  
+    - Great  
+*Other possible options e.g.*  
+    - Imperfect *(only in simple)*  
+    - Imperfect-High *(only in simple simple_plus)*  
+    - Imperfect-Low *(only in simple_plus)*  
+    - Bad
+
+## process-cells:
+`perform:` *Do you want to perform 2.process-cells step? Set to* true *or* false.
+
+`prefilter-features:` *Do you want to use the prefiltering that you created in step 0? Set to* true *or* false.
+
+`output_basedir:` *Name the folder where you would like this step to output files. e.g.* data
+
+`sort_col:` *Column used for sorting data numerically. e.g.* Metadata_Cells_ObjectNumber  
+
+#### merge_columns:
```

Comment:
Is there a reason why "merge_columns` is an H4 heading? I didn't notice it being used before. Can we convert this back to backticks? "`"

Reply:
Oh, maybe b/c there are two levels underneath? For things underneath, we could use bullets. E.g.

`merge_columns:` insert details here
* `image_column:`
* `linking_compartment:`
* `linking_columns:`
    * `cells:`
    * `nuclei:`

raw 👇 
```
`merge_columns:` insert details here
* `image_column:`
* `linking_compartment:`
* `linking_columns:`
    * `cells:`
    * `nuclei:`
```